modular_skyrat/master_files/code/modules/clothing/suits/_suits.dm

```diff
@@ -2,6 +2,12 @@
 	/// Does this object get cropped when worn by a taur on their suit or uniform slot?
 	var/gets_cropped_on_taurs = TRUE
 
+/obj/item/clothing/suit/Initialize(mapload)
+	. = ..()
+	if(islist(allowed))
+		allowed += list(/obj/item/gun,
```

Comment:
This needs to be a list similar to what fits on security armour at minimum, item/gun is far too broad